src/lib/draw/SplitRouteMode.svelte

```diff
@@ -50,13 +51,16 @@
       $map.unproject(e.point).distanceTo($map.unproject(nearbyPoint)) / 1000.0;
 
     // Are we snapped to anything?
-    let candidates = [];
+    let candidates: [number, Position, number][] = [];
```

Comment:
I'd love to have Rust-style wrapper types and say something like [Index, Position, Distance] here, but...